packages/components/modules/notifications/native/NotificationsList/index.tsx

```diff
@@ -69,7 +77,10 @@ const NotificationsList: FC<NotificationsListProps> = ({
 
     return (
       <View>
-        <FlatList data={notifications} renderItem={({ item }) => renderNotificationItem(item)} />
+        <FlatList
+          data={notifications}
+          renderItem={({ item, index }) => renderNotificationItem(item, index)}
```

Comment:
@Ronan-Fernandes could you coordinate this PR with yours (infinite scroller)? we might have conflicts here

Reply:
yeah it is already showing the conflict here since we just merged the infinite scroller PR.

@tsl-ps2 we need to rebase this PR with master and adapt this code to work with the infinite scroller.
since you just added the index there I think this wont be a lot of work